From e9a5ca80d871aab4583df8f5d4a7a83a73e2fb83 Mon Sep 17 00:00:00 2001
From: Simon Busch <morphis@gravedo.de>
Date: Wed, 2 Oct 2013 14:47:29 +0200
Subject: [PATCH] Disable low-level interpreter

It's causing the web process to segfault at:

0  0x00000000 in ?? ()
1  0x414771be in JSC::LLInt::llint_slow_path_resolve_base (exec=0x48a1f038, pc=0x46f5a018)
    at /usr/src/debug/qtwebkit/5.1.0+gitAUTOINC+1e4dd5844b-r0/git/Source/JavaScriptCore/llint/LLIntSlowPaths.cpp:823
2  0x414771be in JSC::LLInt::llint_slow_path_resolve_base (exec=0x48a1f038, pc=0x46f5a018)
    at /usr/src/debug/qtwebkit/5.1.0+gitAUTOINC+1e4dd5844b-r0/git/Source/JavaScriptCore/llint/LLIntSlowPaths.cpp:823

Upstream-Status: Inappropiate [other]

Signed-off-by: Simon Busch <morphis@gravedo.de>
---
 Source/WTF/wtf/Platform.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Source/WTF/wtf/Platform.h b/Source/WTF/wtf/Platform.h
index d61c203..08f89c9 100644
--- a/Source/WTF/wtf/Platform.h
+++ b/Source/WTF/wtf/Platform.h
@@ -743,6 +743,9 @@
 #define ENABLE_LLINT 1
 #endif
 
+#undef ENABLE_LLINT
+#define ENABLE_LLINT 0
+
 #if !defined(ENABLE_DFG_JIT) && ENABLE(JIT) && !COMPILER(MSVC)
 /* Enable the DFG JIT on X86 and X86_64.  Only tested on Mac and GNU/Linux. */
 #if (CPU(X86) || CPU(X86_64)) && (OS(DARWIN) || OS(LINUX))
-- 
1.8.1.2

